metals/src/main/scala/scala/meta/internal/metals/debug/BuildTargetClasses.scala

```diff
@@ -76,9 +76,13 @@ final class BuildTargetClasses(
             .mainClasses(new b.ScalaMainClassesParams(targetsList))
             .map(cacheMainClasses(classes, _))
 
-          val updateTestClasses = connection
-            .testClasses(new b.ScalaTestClassesParams(targetsList))
-            .map(cacheTestClasses(classes, _))
+          // Currently tests are only run using DAP
+          val updateTestClasses =
+            if (connection.isDebuggingProvider)
```

Comment:
```suggestion
            if (connection.isDebuggingProvider || connection.isSbt)
```
I understand `sbt` doesn’t declare this capability so we should have this check here?

Reply:
Ach, good catch, yes!